src/bootstrap/bootstrap.py

```diff
@@ -12,9 +12,38 @@
 import sys
 import tarfile
 import tempfile
+import sqlite3
 
```

Comment:
Maybe the sqlite3 python package is not available on all CI build runners...?

Reply:
I think this particular feature would be fine to skip running if we don't have sqlite3 available. My hope would be that it's mostly available in practice -- so we should still have good support on practical usage, where we might expect parallel invocations.